packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -769,7 +780,11 @@ export class SynapseTable extends React.Component<
                       ></Icon>
                     </span>
                   )}
-                  {isEntityIDColumn && <EntityIDColumnCopyIcon />}
+                  {isEntityIDColumn && (
+                    <span style={{ paddingBottom: '6px' }}>
+                      <EntityIDColumnCopyIcon />
+                    </span>
+                  )}
```

Comment:
This would be way easier if you could pass IconButton props to the EntityIDColumnCopyIcon component, and if that component could update the size of the icon as in the [MUI docs](https://mui.com/material-ui/react-button/#sizes-2). This is really a deficiency in our IconSvg wrapper, so I'll make a PR to address that and then we can rebase this one